### PR TITLE
Move files of couple components into their own component directory

### PR DESCRIPTION
**Pull request by OrkoHunter**
## Hey, I just made a Pull Request!

Clean up the `SortableTable` and `HorizontalScrollGrid` component files.

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/core/src/components/SortableTable/SortableTable.stories.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 
 import React from 'react';
-import { StatusError, StatusOK, StatusWarning } from './Status';
+import { StatusError, StatusOK, StatusWarning } from './../Status';
```
Comment (Rugvip) [inline packages/core/src/components/SortableTable/SortableTable.stories.tsx:18]: Could make this `components/Status`

**Review by Rugvip — APPROVED** [review]
Thanks! :+1:

**Review by Rugvip — APPROVED** [review]
Thanks! :+1:

**Review by Rugvip — APPROVED** [review]
Thanks! :+1:

**Comment by OrkoHunter** [comment]
Checks finally passing. 🎉 No more GitHub 500 errors.

**Comment by Rugvip** [comment]
"Are you sure you want to resubmit this form?"

**Comment by stefanalund** [comment]
I love it when people clean up my code 💯 😄

**Pull request merged**